### PR TITLE
MR-3 Added support to data upload for covariate columns in the data

### PR DESCRIPTION
**Pull request by JanionNevill**
I've not updated the text in the data upload tab(s) because the data upload could still change. I have raised Jira issue MR-10 to updated the instructions at the end of the project, when the upload is finalised.

### CODE REVIEW

**Review by BalancedmonkeY — CHANGES_REQUESTED** [review]
Code looked fine. Tests ran fine. App did not - all main functionality failed to work. After looking at console, gave the following error:
`Warning in rbind("T", "N", "Mean", "SD", c("covar", "group"), c("T", "2",  :
  number of columns of result is not a multiple of vector length (arg 5)`

This indicated to me that it was struggling have a dot in the covariate name (as per design). Once I changed this to an underscore, the app ran fine.

Testing done using ContOutBinCov dataset found with Jira item mr-5.

**Review by BalancedmonkeY — CHANGES_REQUESTED** [review]
All seems to be functioning well.
Only suggestion is to add unit tests for new functions you created.

File: R/DataWrangling.R [hunk 1]
```diff
@@ -39,12 +39,34 @@ CleanData <- function(data) {
   return(dplyr::mutate(data, across(where(is.character), stringr::str_squish)))
 }
 
+#' Create a copy of a data from which does not contain any covariate columns.
+#' 
+#' @param data Data from which to remove covariate columns
+#' @return Data without covariate columns
+RemoveCovariates <- function(data) {
```
[the remark this answers is not in the document]
Comment (JanionNevill) [reply R/DataWrangling.R:46]: Yep, well remembered

**Review by BalancedmonkeY — APPROVED** [review]
Tests looked complete and I've checked that they've passed :D

**Pull request merged**